docs/architecture/architecture-and-security.md

```diff
@@ -0,0 +1,253 @@
+  In principle, it is still possible to create the credentials by deploying a secret for a service account. Such 
+  credentials (the automatically injected token) have the disadvantages to have no restricted lifetime and exist on the 
+  client and server side. Nevertheless, they must be also be rotated (by deleting and recreating the secret). 
+
+In the following we will reference to the names defined here: Gardener-Service-Account, 
+Gardener-Service-Account-Kubeconfig, Shoot-Cluster-Admin-Kubeconfig, Shoot-Cluster-Kubeconfig
+
+### Root Credentials
+
+Credential rotation could not be done completely automatically. There must be some root credentials which has to be
+created and rotated manually by a user with the corresponding permissions. These root credentials are stored in the
+Secret Store.
+
+#### Root Credentials for Gardener Projects
+
+The Gardener-Service-Account-Kubeconfigs for the Gardener-LaaS-Project and the Gardener-Resource-Cluster-Project 
+are root credentials for a LaaS landscape. These must be created and rotated manually by a user with 
+`Service Account Manager` role.
+
+The corresponding Gardener-Service-Accounts must have the `Admin` role, but not the `Service Account Manager` role. 
+This prevents to request new tokens and is a Gardener recommendation. If a service account is allowed to request 
+new tokens and one such token leaks, an attacker could create new tokens forever and rotation could never invalidate 
+them, except the corresponding service account is replaced.
+
+#### Root Credentials for GCP Projects
+
+The Gardener shoot cluster custom resources in the Gardener-LaaS-Project and the Gardener-Resource-Cluster-Project 
+references secrets containing the secret key of a [GCP](https://cloud.google.com/) service account. This is required 
+for the creation and maintenance of shoot clusters by the Gardener. It is recommended to use one secret in both Garden 
+projects each containing the secret key of a service account of a different GCP project.
+
+The secret keys for the GCP service account are root credentials of a LaaS landscape. The secret keys must be created 
+and rotated manually by a human, and stored and replaced in the corresponding k8s secrets of the Gardener-LaaS-Project 
+and the Gardener-Resource-Cluster-Project by the Deploy-Pipeline.
+
+#### Image Pull Secrets
+
+Image pull secrets are another group of root credentials stored in the Secret Store which must be rotated manually. 
+
+### Deploy-Pipeline
+
+This Chapter describes which credentials are created and rotated with every execution by the Deploy-Pipeline.
+
+**Credentials for GCP Project:** The Deploy-Pipeline must update the k8s secrets stored in the Gardener projects 
+Gardener-LaaS-Project and Gardener-Resource-Cluster-Project used to create and maintain shoot clusters. Therefore,
+it must store there the latest root credentials for GCP Projects stored in the Secret Store. The update of the k8s secrets 
```

Comment:
This sentence sounds a little bit "sperrig". Maybe you can rephrase that?

Reply:
If have tried is :-)